lib/jxl/decode.cc

```diff
@@ -438,10 +438,15 @@ struct JxlDecoderStruct {
   size_t preview_out_size;
   size_t image_out_size;
 
-  // TODO(lode): merge these?
   JxlPixelFormat preview_out_format;
   JxlPixelFormat image_out_format;
 
+  // For extra channels
+  std::vector<bool> extra_channel_wanted;  // Empty if none, reset each frame
```

Comment:
Can we avoid vector<bool> here?

Reply:
You mean for all or only the bool one? What do you propose instead?

Reply:
uh, github ate my answer :P `vector<bool>`

Reply:
Done, removed